pkg/ebpf/processor_funcs.go

```diff
@@ -308,13 +214,7 @@ func (t *Tracee) processSchedProcessExec(event *trace.Event) error {
 	return nil
 }
 
-func (t *Tracee) processSchedProcessFork(event *trace.Event) error {
-	return t.convertArgMonotonicToEpochTime(event, "start_time")
-}
```

Comment:
I want to get the opinion of @roikol  or @AlonZivony before removing that

Reply:
I will write it also here - I am against removing it and event for adding new time arguments to the functionality.

Reply:
https://github.com/aquasecurity/tracee/pull/3498#discussion_r1337884152